playwright-e2e/config/.env.dev

```diff
@@ -0,0 +1,17 @@
+# Ensure environment variable names are unique across different apps
+
+# Singular
+singularBaseURL=https://singular.dev.datadonationplatform.org
```

Comment:
Question: I think we should rename all env variable names to uppper-case with underscore instead camel-case, like `SINGULAR_BASE_URL`. Thoughts? Objection?

Reply:
no objections

Reply:
Great. I'll make change in this PR.